_projects/antlr_v4.md

```diff
@@ -19,7 +19,8 @@ requirements:
   - "Being familiar with the current Bear API and what it provides."
   - "Knowledge of ANTLR and Java is a bonus."
   - "Knowledge of ASTs and basic language grammar."
-status: in_progress
+status:
```

Comment:
I do not believe this is necessary.

There are still many other .md files in this directory which use `status: in_progress` so it is extremely unlikely that only this one should be modified.

Ideally , none are modified in this PR.